src/wallet/wallet.cpp

```diff
@@ -3076,7 +3076,7 @@ bool CWallet::CommitTransaction(CTransactionRef tx, mapValue_t mapValue, std::ve
         wtxNew.fTimeReceivedIsTxTime = true;
         wtxNew.fFromMe = true;
 
-        LogPrintf("CommitTransaction:\n%s", wtxNew.tx->ToString());
+        LogPrintf("CommitTransaction:\n%s\n", wtxNew.tx->ToString());
```

Comment:
Unfortunately this change is wrong, as tx->ToString() adds a final newline: https://github.com/bitcoin/bitcoin/blob/master/src/primitives/transaction.cpp#L99

Reply:
ah, well spotted. Now fixed.

Incidentally, I hate this log. As far as I'm aware it's the only one with newlines embedded and it confuses my log parser!